lib/moonchild.js

```diff
@@ -3,12 +3,17 @@
 var _ = require('underscore'),
     parser = require('./metadata'),
     estraverse = require('estraverse'),
-    expanders = require('../third_party/expanders');
+    expanders = require('../third_party/expanders'),
+    util = require('./util.js'),
+    createChannel = require('./channel.js').createChannel;
```

Comment:
If you make the channel more OO as I suggested, this should become `Channel = require('./channel')`. (Also, leave off '.js' to be consistent with the other require statements.